diff -urNad xrdp-0.9.24-orig/sesman/sesman.ini xrdp-0.9.24/sesman/sesman.ini
--- xrdp-0.9.24-orig/sesman/sesman.ini	2023-12-30 17:34:25.000000000 +0200
+++ xrdp-0.9.24/sesman/sesman.ini	2024-01-28 14:43:06.885402484 +0200
@@ -12,7 +12,7 @@
 ReconnectScript=reconnectwm.sh
 
 [Security]
-AllowRootLogin=true
+AllowRootLogin=false
 MaxLoginRetry=4
 TerminalServerUsers=tsusers
 TerminalServerAdmins=tsadmins
@@ -156,7 +156,7 @@
 ; Note: Log levels can be any of: core, error, warning, info, debug, or trace
 LogLevel=INFO
 EnableSyslog=true
-#SyslogLevel=INFO
+SyslogLevel=INFO
 #EnableConsole=false
 #ConsoleLevel=INFO
 #EnableProcessId=false
diff -urNad xrdp-0.9.24-orig/xrdp/xrdp.ini xrdp-0.9.24/xrdp/xrdp.ini
--- xrdp-0.9.24-orig/xrdp/xrdp.ini	2023-12-30 17:34:26.000000000 +0200
+++ xrdp-0.9.24/xrdp/xrdp.ini	2024-01-28 14:43:07.254403681 +0200
@@ -181,7 +181,7 @@
 LogFile=xrdp.log
 LogLevel=INFO
 EnableSyslog=true
-#SyslogLevel=INFO
+SyslogLevel=INFO
 #EnableConsole=false
 #ConsoleLevel=INFO
 #EnableProcessId=false
