diff -urNad xrdp-0.10.2-orig/sesman/sesman.ini.in xrdp-0.10.2/sesman/sesman.ini.in
--- xrdp-0.10.2-orig/sesman/sesman.ini.in	2024-12-24 15:09:57.000000000 +0200
+++ xrdp-0.10.2/sesman/sesman.ini.in	2025-01-04 11:42:33.439367738 +0200
@@ -12,7 +12,7 @@
 ReconnectScript=reconnectwm.sh
 
 [Security]
-AllowRootLogin=true
+AllowRootLogin=false
 MaxLoginRetry=4
 TerminalServerUsers=tsusers
 TerminalServerAdmins=tsadmins
@@ -115,7 +115,7 @@
 LogFile=xrdp-sesman.log
 LogLevel=INFO
 EnableSyslog=true
-#SyslogLevel=INFO
+SyslogLevel=INFO
 #EnableConsole=false
 #ConsoleLevel=INFO
 #EnableProcessId=false
diff -urNad xrdp-0.10.2-orig/xrdp/xrdp.ini.in xrdp-0.10.2/xrdp/xrdp.ini.in
--- xrdp-0.10.2-orig/xrdp/xrdp.ini.in	2024-12-24 15:09:57.000000000 +0200
+++ xrdp-0.10.2/xrdp/xrdp.ini.in	2025-01-04 11:43:04.787562193 +0200
@@ -205,7 +205,7 @@
 LogFile=xrdp.log
 LogLevel=INFO
 EnableSyslog=true
-#SyslogLevel=INFO
+SyslogLevel=INFO
 #EnableConsole=false
 #ConsoleLevel=INFO
 #EnableProcessId=false
