diff -urNad xrdp-0.9.23.1-orig/sesman/sesman.ini xrdp-0.9.23.1/sesman/sesman.ini
--- xrdp-0.9.23.1-orig/sesman/sesman.ini	2023-09-27 21:00:00.000000000 +0300
+++ xrdp-0.9.23.1/sesman/sesman.ini	2023-10-01 15:00:27.483152567 +0300
@@ -12,7 +12,7 @@
 ReconnectScript=reconnectwm.sh
 
 [Security]
-AllowRootLogin=true
+AllowRootLogin=false
 MaxLoginRetry=4
 TerminalServerUsers=tsusers
 TerminalServerAdmins=tsadmins
@@ -156,7 +156,7 @@
 ; Note: Log levels can be any of: core, error, warning, info, debug, or trace
 LogLevel=INFO
 EnableSyslog=true
-#SyslogLevel=INFO
+SyslogLevel=INFO
 #EnableConsole=false
 #ConsoleLevel=INFO
 #EnableProcessId=false
diff -urNad xrdp-0.9.23.1-orig/xrdp/xrdp.ini xrdp-0.9.23.1/xrdp/xrdp.ini
--- xrdp-0.9.23.1-orig/xrdp/xrdp.ini	2023-09-27 21:00:00.000000000 +0300
+++ xrdp-0.9.23.1/xrdp/xrdp.ini	2023-10-01 15:00:27.609153059 +0300
@@ -181,7 +181,7 @@
 LogFile=xrdp.log
 LogLevel=INFO
 EnableSyslog=true
-#SyslogLevel=INFO
+SyslogLevel=INFO
 #EnableConsole=false
 #ConsoleLevel=INFO
 #EnableProcessId=false
