diff -urNad xrdp-0.10.4.1-orig/sesman/sesman.ini.in xrdp-0.10.4.1/sesman/sesman.ini.in
--- xrdp-0.10.4.1-orig/sesman/sesman.ini.in	2025-07-07 15:37:24.000000000 +0300
+++ xrdp-0.10.4.1/sesman/sesman.ini.in	2025-08-10 11:05:33.114862923 +0300
@@ -12,7 +12,7 @@
 ReconnectScript=reconnectwm.sh
 
 [Security]
-AllowRootLogin=true
+AllowRootLogin=false
 MaxLoginRetry=4
 TerminalServerUsers=tsusers
 TerminalServerAdmins=tsadmins
@@ -115,7 +115,7 @@
 LogFile=xrdp-sesman.log
 LogLevel=INFO
 EnableSyslog=true
-#SyslogLevel=INFO
+SyslogLevel=INFO
 #EnableConsole=false
 #ConsoleLevel=INFO
 #EnableProcessId=false
diff -urNad xrdp-0.10.4.1-orig/xrdp/xrdp.ini.in xrdp-0.10.4.1/xrdp/xrdp.ini.in
--- xrdp-0.10.4.1-orig/xrdp/xrdp.ini.in	2025-07-07 15:37:24.000000000 +0300
+++ xrdp-0.10.4.1/xrdp/xrdp.ini.in	2025-08-10 11:05:33.190863212 +0300
@@ -204,7 +204,7 @@
 LogFile=xrdp.log
 LogLevel=INFO
 EnableSyslog=true
-#SyslogLevel=INFO
+SyslogLevel=INFO
 #EnableConsole=false
 #ConsoleLevel=INFO
 #EnableProcessId=false
